content/tracing/advanced_usage/custom_tagging.md

```diff
@@ -44,6 +44,31 @@ class ServletImpl extends AbstractHttpServlet {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+**Adding tags to a span**
```

Comment:
Missing the part where you explain how to tag the active span: https://github.com/DataDog/documentation/pull/3025/files#diff-aba644c513074d732c91383fee345d06R63